src/Worker.Extensions.DurableTask/DefaultDurableClientContext.cs

```diff
@@ -98,7 +98,13 @@ public ValueTask<ConversionResult> ConvertAsync(ConverterContext converterContex
             try
             {
                 DurableTaskGrpcClient.Builder builder = DurableTaskGrpcClient.CreateBuilder();
-                builder.UseAddress(inputData.rpcBaseUrl);
+                if (!Uri.TryCreate(inputData.rpcBaseUrl, UriKind.Absolute, out Uri? baseUriResult))
+                {
+                    InvalidOperationException exception = new($"Failed to parse the {nameof(inputData.rpcBaseUrl)} field from the input binding data.");
+                    return new ValueTask<ConversionResult>(ConversionResult.Failed(exception));
+                }
```

Comment:
Not strictly required, but improving some validation based on some bugs I ran into during local development.